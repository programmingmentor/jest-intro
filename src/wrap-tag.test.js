import wrapTag from './wrap-tag';

test('if it wraps tag', () => {
    const wrappedCode = wrapTag('Hello World', 'h1');
    // expect(wrappedCode).toBe('<h1>Hello World</h1>');
    expect(wrappedCode).toMatchSnapshot();
});