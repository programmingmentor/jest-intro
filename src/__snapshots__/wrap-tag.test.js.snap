// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if it wraps tag 1`] = `"<h1>Hello World</h1>"`;
